packages/core/lib/generators/assets_generator.dart

```diff
@@ -99,14 +102,20 @@ String generateAssets(
 List<String> _getAssetRelativePathList(
   String rootPath,
   List<String> assets,
+  List<String> excludes,
 ) {
   final assetRelativePathList = <String>[];
   for (final assetName in assets) {
     final assetAbsolutePath = join(rootPath, assetName);
+    if (excludes.any((exclude) => assetAbsolutePath.contains(exclude))) {
```

Comment:
How about making `exclude` to support the `glob` pattern which can be more convenient?
https://pub.dev/packages/glob